PKGBUILD

```diff
@@ -72,7 +72,10 @@ prepare() {
   rustup component add rust-src
 
   echo "Installing rust-bindgen"
-  cargo install --locked --version $(scripts/min-tool-version.sh bindgen) bindgen
+  # uninstall potentially installed previous versions of bindgen which
+  # contained the binaries which are now in bindgen-cli
+  cargo uninstall bindgen
+  cargo install --locked --version 0.62.0 bindgen-cli
```

Comment:
The older bindgen version didn't work with LLVM 16 which is now in the ArchLinux repos. See also
https://github.com/Rust-for-Linux/linux/commit/08ab786556ff177086ce93b26daf2a58edd10968

0.62.0 seems to work without any changes to the code necessary.